scripts/py_matter_yamltests/matter_yamltests/websocket_runner.py

```diff
@@ -0,0 +1,105 @@
+#
```

Comment:
I have only spent a couple minute reviewing, so it is possible if I spend more time I will fully understand:

Why is the web-socket stuff inside matter_yamltests? Is this going to be used for anything else other than the chip-tool version?